tool/jt.rb

```diff
@@ -2235,7 +2237,10 @@ def rubocop(*args)
       env = { 'GEM_HOME' => gem_home, 'GEM_PATH' => gem_home }
       sh env, 'ruby', "#{gem_home}/bin/rubocop", *args
     else
-      sh 'rubocop', '_0.66.0_', *args
+      unless sh('rubocop', "_#{RUBOCOP_VERSION}_", *args, continue_on_failure: true)
```

Comment:
This will try to install RuboCop also simply when it's installed but the command fails.
Probably better to first run some command to check if it's installed.

I can already merge this though.

Reply:
Probably a good solution for that is to have a small rubocop wrapper, e.g. in `tool/rubocop` that tries `gem 'rubocop', '0.66.0'` and if that's a LoadError, then `gem install` and `exec` to itself.

Reply:
I didn't want to run a RubyGems command or app before very invocation, as that seems slow, but could do.

Reply:
Yeah, that wrapper idea would avoid that. Basically the wrapper would be similar to `cat $(which rubocop)`, but would also auto-install + re-exec if not installed yet.